bitbucket-slack-server-integration-plugin/src/main/java/com/atlassian/bitbucket/plugins/slack/listener/BitbucketNotificationEventListener.java

```diff
@@ -114,18 +130,23 @@ public void onEvent(final PullRequestEvent event) {
         });
     }
 
+    private Set<ApplicationUser> participantsToApplicationUser(Set<PullRequestParticipant> participants) {
+        return participants.stream()
+                .map(PullRequestParticipant::getUser)
+                .collect(Collectors.toSet());
+    }
+
     private Set<ApplicationUser> getUsersAddedToPullRequest(final PullRequestEvent event) {
-        if (event instanceof PullRequestReviewersUpdatedEvent) {
-            final PullRequestReviewersUpdatedEvent e = (PullRequestReviewersUpdatedEvent) event;
-            return e.getAddedReviewers();
+        if (event instanceof PullRequestOpenedEvent) {
+            final PullRequestOpenedEvent e = (PullRequestOpenedEvent) event;
+            return participantsToApplicationUser(e.getPullRequest().getReviewers());
         }
```

Comment:
So, before the change the method worked with `PullRequestReviewersUpdatedEvent` event only, after update - it works with `PullRequestOpenedEvent` class only. I suppose, it would be valuable to make the method working for both events.